infer_structcol/main.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import numpy as np 
+
+class Spectrum(pd.DataFrame):
+    def __init__(self, reflectance_file, transmission_file=None)
+        #pd.read_table or similar
+
+        if transmission_file is not None:
+            raise NotImplementedError()
+
+    @property
+    def wavelength(self):
+        return self['wavelength'].values
+    @property
+    def meas(self):
+        return self['measurement'].values
+    @property
+    def sigma(self):
+        return self['sigma'].values
+
+class Sample:
+    def __init__(self, particlesize, thickness, particleindex, matrixindex, mediumindex=1, incidentangle=0):
+        self.particlesize = particlesize # can we do something clever here with units? maybe using pint?
```

Comment:
Yeah, I think the user should define the variable particlesize as a pint variable, so the class should expect pint variables